hawc/apps/hawc_admin/serializers.py

```diff
@@ -0,0 +1,10 @@
+from hawc.apps.assessment.models import Values
+from hawc.apps.common.helper import FlatExport
+
+
+class ValuesExportOptions:
```

Comment:
i'd move this to the `assesssment.exports`; there are other common patterns you can check that follow this